### DISCUSSION TITLE
Lineage Boot Animation never ends.

### DISCUSSION THREAD

**Discussion by laurenslo** [pgale/lineage_15.1_Installation_TSV#22] [General]
As the title says. I have tried to flash lineage-15.1-20240602-UNOFFICIAL-starfire multiple times, but I can't get past the lineage boot animation. It goes forever and doesn't boot all the way. I followed the steps to the best of my ability, but I am stuck. I re-flashed kingston a11 without issue. Any help would be appreciated.

**Comment by pgale** [comment]
I wonder if it's something to do with partition sizes. I have seen similar when I was trying all kinds of ROM's including A11. 

I would try flashing one of the stock ROMs - maybe 084. Or you could also try running the Lenovo recovery app which resets and installs a stock ROM. 

Then try again...

**Comment by laurenslo** [comment]
That worked perfectly. Your work on this is really great. Thank you very much for everything!

**Reply by pgale** [reply]
Great. Enjoy using it!
